locust/configuration.py

```diff
@@ -20,3 +20,13 @@ def write_file(stringJSON):
     except Exception as err:
         logger.info(err)
         return False, err
+
+def read_JSON():
```

Comment:
what is the different with `read_file()`

Reply:
read_file using file.read returning whole string, used for textarea
read_JSON using json.load returning json string, easier to use for parse data :D